packages/snaps-jest/src/internals/simulation/store/ui.ts

```diff
@@ -1,11 +1,10 @@
-import type { DialogType } from '@metamask/snaps-sdk';
 import type { PayloadAction } from '@reduxjs/toolkit';
 import { createAction, createSelector, createSlice } from '@reduxjs/toolkit';
 
 import type { ApplicationState } from './store';
 
 export type Interface = {
-  type: DialogType;
+  type: string;
```

Comment:
@Mrtenz This type is only used internally, right?

Reply:
I think these are internal only, I convert them to the Dialog type in `getInterfaceResponse`